### ISSUE TITLE
Test: add ACL tests for each app

### ISSUE THREAD

**Issue by sohkai** [aragon/aragon-apps#685]
From https://github.com/aragon/aragon-apps/pull/611#issuecomment-465090647:

> * Verify that all externally-accessible, state-changing functionality is protected by authentication and the ACL system is set-up correctly. Consider adding ACL/permission specific unittests. add pos/negative checks to verify that only permitted accounts are allowed to call privileged methods.
> * e.g. see `survey.sol` below; unittests are set-up to grant the permissions `CREATE_SURVEYS_ROLE`, `MODIFY_PARTICIPATION_ROLE` to `ANY_ENTITY` essentially allowing anyone to perform authenticated actions not testing if the app specific ACL configuration is set up correctly.

**Comment by haythem96** [comment]
is this still relevant ? I can work on it...

**Comment by sohkai** [comment]
@haythem96 Yes, this is still relevant! We have improved the testing on Voting and Payroll to have slightly better permission tests, but most, if not all, of the apps can use more tests for this :).

There are additionally some more _interesting_ tests for each role’s parameterization, but I wouldn’t work on testing this aspect of the ACL just yet since it’s really cumbersome to do without a framework helper.

**Comment by haythem96** [comment]
@sohkai let's say for the case of the survey app, should I in some tests restrict permission to a specific address and test pos/negative check to verify only that specific address is permitted ? as in that app it grant permission to `ANY_ENTITY`

**Comment by sohkai** [comment]
@haythem96 Yes, that would be the correct idea.
